parsec/backend/memory/pki.py

```diff
@@ -107,6 +109,16 @@ async def submit(
                 # last one represent the current state of this x509 certificate.
                 break
 
+        # Optional check for client compatibility with version < 2.8.3
+        if submitter_der_x509_certificate_email:
+            # Assert email not used.
+            _, total = await self._user_component.find_humans(
```

Comment:
would be better to use sync calls only here (for instance by introducing a `MemoryUserComponent._find_humans` sync method): this is allow us to have atomic operation given we don't have the for update locking and transaction that is available in postgresql